freqtrade/optimize/__init__.py

```diff
@@ -15,6 +15,63 @@
 logger = logging.getLogger(__name__)
 
 
+def sanitize_start_stop_date(tickerlist: List[Dict], timerange, pair):
```

Comment:
could you please add a type-hint on return type 
And a test (or a few tests) testing all cases this function can run into

Reply:
I'll take a look at this, very new to python so its a good learning opportunity